### PR TITLE
Fix studio to honor airlock and local vm dev

### PR DESCRIPTION
**Pull request by elliott-davis**
tenor-19422830
Signed-off-by: Travis Elliott Davis <edavis@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/studio/bin/hab-studio.sh [hunk 1]
```diff
@@ -420,7 +420,11 @@ new_studio() {
       $bb mount $v -t proc proc $HAB_STUDIO_ROOT/proc
     fi
     if ! $bb mount | $bb grep -q "on $HAB_STUDIO_ROOT/sys type"; then
-      $bb mount $v --rbind /sys $HAB_STUDIO_ROOT/sys
+      if [ -z "${KRANGSCHNAK+x}" ]; then
```
Comment (raskchanky) [inline components/studio/bin/hab-studio.sh]: What's the `+x` do?

Comment (elliott-davis) [reply components/studio/bin/hab-studio.sh]: Checks for string rather than just is set/not set. `-z` checks for non-zero length.

**Comment by chefsalim** [comment]
tenor-173475660

**Comment by elliott-davis** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/303707179).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/303707179).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**